src/auro-datetime.js

```diff
@@ -176,7 +151,18 @@ class AuroDatetime extends LitElement {
    * @returns {string} - time string
    */
   humanTime() {
-    const newTime = new Date();
+    let newTime = new Date();
+
+    if (this.utc) {
+      this.timeTemplate.timeZone = 'UTC';
+      newTime = new Date(this.utc);
+
+      return newTime.toLocaleString('en-us', this.timeTemplate).replace(/^0+/u, '').toLowerCase();
+    } else if (this.setDate) {
+      newTime = new Date(this.setDate);
+
+      return newTime.toLocaleString('en-us', this.timeTemplate).replace(/^0+/u, '').toLowerCase();
+    }
```

Comment:
We can use the one return at the end of the function.

```suggestion
    if (this.utc) {
      this.timeTemplate.timeZone = 'UTC';
      newTime = new Date(this.utc);
    } else if (this.setDate) {
      newTime = new Date(this.setDate);
    }
```